Dockerfile

```diff
@@ -1,14 +1,18 @@
-FROM python:2.7
+FROM python:2.7.12-alpine
+
+RUN apk add --update --no-cache alpine-sdk wget git
```

Comment:
@th3w4y do you think we still need git and wget? If not we should remove it to keep it clean.